### DISCUSSION TITLE
SOCP Certification

### DISCUSSION THREAD

**Discussion by bvvard** [Security-Onion-Solutions/securityonion#8661] [General] [answered]
Hi all,

New to the security onion world. My company is sponsoring me taking the paid training offered by SOS to ultimately acquire the certification. Haven't found much pertaining to the certification or people who have it in general. Would love some insight on the cert. Thanks! Pleased to share I now have our production environment deployed here at my company.
#SOCP #FNG #Training

**Comment by reyesj2** [comment, marked as answer]
SOCP here,

I attended the in person training "Security Onion 2 Fundamentals for Analysts & Admins".  A good overview of Security Onion. With great instructors that helped breakdown or expand upon topics discussed to make sure everyone had a chance to understand the topics. They did really good at avoiding death by PowerPoint by incorporating hands-on labs surrounding the current topic. 

As for the certification, after the course you will leave with three licensed training materials (Lab workbook, SO fundamentals for analysts & administrators, and a copy of the current SO paperback documentation). All of which you are authorized to bring into the testing center during your exam for you to reference. With the documentation & course lessons you are well on your way to passing the SOCP.
